ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -755,39 +757,27 @@ define([
       }
     },
 
-    // Convert Lat-Lon to UTM coordinates. Returns undefined if lat or lon is undefined or not a number.
+    // Convert Lat-Lon to UTM/UPS coordinates. Returns undefined if lat or lon is undefined or not a number.
     // Returns undefined if the underlying call to usng fails. Otherwise, returns an object with:
     //
     //   easting    : FLOAT
     //   northing   : FLOAT
-    //   zoneNumber : INTEGER (>=1 and <= 60)
+    //   zoneNumber : INTEGER (>=0 and <= 60)
     //   hemisphere : STRING (NORTHERN or SOUTHERN)
-    LLtoUTM: function(lat, lon) {
+    LLtoUtmUps: function(lat, lon) {
       if (isNaN(lat) || isNaN(lon)) {
         return undefined
       }
-
-      var utmCoords = []
-      var converterReturnCode = converter.LLtoUTM(lat, lon, utmCoords)
-
-      if (converterReturnCode === 'undefined') {
-        return undefined
-      }
-
-      var results = {}
-      results.easting = utmCoords[0]
```

Comment:
These lines may need to be modified based on https://github.com/codice/usng.js/issues/19